### PR TITLE
docs: Add a troubleshooting guide for CIDR pre-check failure

### LINKED ISSUE #3235: Add troubleshooting note to deal with Service CIDR pre check failure
**Component**:

'doc'

**Why this is needed**:

When `bootstrap.sh` fails because of Service CIDR pre-check fails: https://github.com/scality/metalk8s/discussions/3233, the administrator doesn't have much info/explanations to deal with it and fix the platform to install bootstrap node.

**What should be done**:

We should add a troubleshooting node in https://metal-k8s.readthedocs.io/en/2.8.0/installation/troubleshooting/bootstrap_installation_errors.html to explain to the administrator how to set up the route that is required for bootstrap node install.

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by alexandre-allard**
Closes: #3235

### CODE REVIEW

File: docs/installation/troubleshooting/bootstrap_installation_errors.rst [hunk 1]
```diff
@@ -34,8 +36,134 @@ salt-master container) do not start, perform the following checks:
...
+This error happens if there is no route matching this network CIDR and no
+default route configured.
+
+You can solve this issue by either adding a default route to your host or
+adding a dummy network interface used to define a route for this network.
+
+Configuring a default route:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To configure a default route, please, refer to the official documentation of
```
Comment (gdemonet) [inline docs/installation/troubleshooting/bootstrap_installation_errors.rst]:
```suggestion
To configure a default route, refer to the official documentation of
```

File: docs/installation/troubleshooting/bootstrap_installation_errors.rst [hunk 2]
```diff
@@ -34,8 +36,134 @@ salt-master container) do not start, perform the following checks:
...
+     EOF
+
+     chmod +x /etc/sysconfig/network-scripts/ifdown-dummy
+
+Create the ``route-dummy-metalk8s`` network script:
+
+  .. code-block:: shell
+
+     cat > /etc/sysconfig/network-scripts/route-dummy-metalk8s << EOF
+     $(salt-call pillar.get networks:service --out=txt | cut -d' ' -f2-) dev dummy-metalk8s
```
Comment (gdemonet) [inline docs/installation/troubleshooting/bootstrap_installation_errors.rst]:
```suggestion
     $(salt-call --local pillar.get networks:service --out=txt | cut -d' ' -f2-) dev dummy-metalk8s
```

File: docs/installation/troubleshooting/bootstrap_installation_errors.rst [hunk 3]
```diff
@@ -34,8 +36,134 @@ salt-master container) do not start, perform the following checks:
...
+
+     ifup dummy-metalk8s
+
+**CentOS / RHEL 8 (and other NetworkManager based dists)**
+
+Retrieve the service network CIDR:
+
+  .. code-block:: shell
+
+     salt-call pillar.get networks:service --out=txt | cut -d' ' -f2-
```
Comment (gdemonet) [inline docs/installation/troubleshooting/bootstrap_installation_errors.rst]:
```suggestion
     salt-call --local pillar.get networks:service --out=txt | cut -d' ' -f2-
```

File: docs/installation/troubleshooting/bootstrap_installation_errors.rst [hunk 4]
```diff
@@ -34,8 +36,134 @@ salt-master container) do not start, perform the following checks:
...
+Create the ``dummy-metalk8s`` interface:
+
+  .. code-block:: shell
+
+     nmcli connection add type dummy ifname dummy-metalk8s ipv4.method manual ipv4.addresses <dummy-iface-ip> ipv4.routes <network-cidr>
+
+  .. note::
+
+     Replace ``<dummy-iface-ip>`` by any available IP in the previously
+     retrieved network CIDR (e.g. 10.96.0.1 for a 10.96.0.0/12 network CIDR)
```
Comment (gdemonet) [inline docs/installation/troubleshooting/bootstrap_installation_errors.rst]: not 10.96.0.1, as it looks like it's conflicting with `kubernetes.default.svc.cluster.local`

**Review by gdemonet — CHANGES_REQUESTED** [review]

File: docs/installation/troubleshooting/bootstrap_installation_errors.rst (hunk 1 above)
Comment (alexandre-allard) [reply docs/installation/troubleshooting/bootstrap_installation_errors.rst]: done

File: docs/installation/troubleshooting/bootstrap_installation_errors.rst (hunk 2 above)
Comment (alexandre-allard) [reply docs/installation/troubleshooting/bootstrap_installation_errors.rst]: done

File: docs/installation/troubleshooting/bootstrap_installation_errors.rst (hunk 3 above)
Comment (alexandre-allard) [reply docs/installation/troubleshooting/bootstrap_installation_errors.rst]: done

File: docs/installation/troubleshooting/bootstrap_installation_errors.rst (hunk 4 above)
Comment (alexandre-allard) [reply docs/installation/troubleshooting/bootstrap_installation_errors.rst]: done

**Review by gdemonet — APPROVED** [review]

**Pull request merged**